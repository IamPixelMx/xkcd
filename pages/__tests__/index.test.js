import React from 'react';
import configureStore from 'redux-mock-store';
import { renderWithProviderSnapshot } from 'jest-utils';
import Home from '../';

const mockStore = configureStore([]);

describe('Home', () => {
  const homeElement = renderWithProviderSnapshot(<Home />, { mockStore }).toJSON();

  it('Renders Home page', () => {
    expect(homeElement).toMatchSnapshot();
  });
});
