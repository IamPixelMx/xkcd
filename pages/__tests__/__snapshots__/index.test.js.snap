// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Renders Home page 1`] = `
<main
  className="container"
>
  <div
    className="card"
  >
    <div
      className="card-content"
    >
      <span>
        <p
          className="title"
        />
      </span>
      <time
        dateTime="undefined-undefined-undefined"
      >
        <small>
          Invalid Date
        </small>
      </time>
    </div>
    <div
      className="card-image"
    >
      <figure
        className="image comic"
      >
        <img
          alt="comic-image"
        />
      </figure>
    </div>
    <div
      className="card-content has-text-centered mb-2"
    >
      <p
        className="subtitle"
      />
    </div>
    <footer
      className="card-footer"
    >
      <p
        className="card-footer-item"
      >
        <a
          aria-label="search-input"
          onClick={[Function]}
        >
          Agregar a Favoritos ‚ù§
        </a>
      </p>
    </footer>
  </div>
  <nav
    aria-label="pagination"
    className="pagination column is-rounded"
    role="navigation"
  >
    <button
      className="pagination-previous button is-dark"
      name="prev"
      onClick={[Function]}
    >
      Siguiente
    </button>
    <button
      className="pagination-next button is-dark"
      name="next"
      onClick={[Function]}
    >
      Anterior
    </button>
  </nav>
</main>
`;
